### PR TITLE
Invalidate 'Wrong Type' snmp_walk() data

### PR DESCRIPTION
**Pull request by josephtingiris**
Certain, invalid, values must not be passed by snmp_walk() or they will end up being discovered as present when they are, in fact, not present (e.g. Temperature, Fan, & Voltage sensors).

This patch adds 'Wrong Type' to the existing snmp_walk() preg_match() condition to invalidate the $data.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by josephtingiris** [comment]
These stop appearing on device pages ...

image

File: includes/snmp.inc.php [hunk 1]
```diff
@@ -386,7 +386,9 @@ function snmp_walk($device, $oid, $options = null, $mib = null, $mibdir = null)
     $data = str_replace('"', '', $data);
     $data = str_replace('End of MIB', '', $data);
 
-    if (is_string($data) && (preg_match('/No Such (Object|Instance)/i', $data))) {
+
+    if (is_string($data) && (preg_match('/No Such (Object|Instance)/i', $data) || preg_match('/Wrong Type(.*)should be/',$data))) {
```
Comment (SourceDoctor) [inline includes/snmp.inc.php]: could you remove empty line before this line
and change `,$data`  to `, $data`


>

Comment (josephtingiris) [reply includes/snmp.inc.php]: Thanks.  done.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-61-release-changelog-february-2020/11145/1

**Pull request merged**